composer.json

```diff
@@ -15,13 +15,13 @@
         }
     },
     "require": {
-        "php": ">=7.0",
-        "symfony/console": "~3.0|~4.0|~5.0",
+        "php": ">=7.4",
+        "symfony/console": "~3.0|~4.0|~5.0|~6.0",
         "php-di/invoker": "~2.0",
         "psr/container": "^1.0|^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.4",
+        "phpunit/phpunit": ">=6.4",
```

Comment:
This runs the risk of breaking with new PHPUnit versions. Maybe try something like `^6.4 || 7 || 8`?